cmd/activator/main.go

```diff
@@ -57,7 +57,7 @@ import (
 	activatorhandler "knative.dev/serving/pkg/activator/handler"
 	activatornet "knative.dev/serving/pkg/activator/net"
 	"knative.dev/serving/pkg/apis/networking"
-	"knative.dev/serving/pkg/autoscaler"
+	autoscalerMetric "knative.dev/serving/pkg/autoscaler/metrics"
```

Comment:
it would be more aligned with conventions if the package alias is all lowercase

Reply:
+1 and plural to match the package name `metrics`

```suggestion
	autoscalermetrics "knative.dev/serving/pkg/autoscaler/metrics"
```

Reply:
And to make it shortes `asmetrics` should even be better.